backend/routes/trainer.routes.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+
+
+const trainerController = require('../controllers/trainer.controller');
+
+// Retrieve all trainers
+router.get('/trainers', trainerController.getTrainers);
+
+// Get single trainer
+router.get('/trainers/:trainerID', trainerController.getTrainer);
+
+//get approve status of trainer
+router.get('/trainers/:trainerID/approve', trainerController.getApproveStatus);

```

Comment:
This route is the same as updateApproveStatus one.
Please change it so that it does not cause confusion.
`/getStatusOfTrainer` then pass trainerId as query params